.github/ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,21 @@
 ## Description
-Add a brief and meaningful description.
+<!-- Add a brief and meaningful description. -->
 
 ## Expected Behavior
+<!-- Describe the expected behaviour. -->
 
 ## Actual Behavior
+<!-- Describe the current/actual behaviour. -->
 
-## Steps to Reproduce
-1.
-2.
+## Environment
 
-## Specifications
-  - Browser:
-  - Browser version:
-  - OS: 
+* Operating system: (E.g RHEL 7.6 )
+* Browser and version: (E.g Google Chrome Version 70.0.3538.102 (Official Build) (64-bit))
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Yarn version:
```

Comment:
Hi @mindreeper2420,

Is it making sense now? Should we remove the OCP too?

Reply:
Hi @camilamacedo86 - I’d remove OCP. Perhaps replace with node version? I like the addition of asking the yarn version.

Reply:
Sure, but according to the Readme, development is not done using OCP. If people using the production Integr8ly environment will log bugs here, then I’m fine with leaving the OCP piece.